server/app/views/admin/apikeys/ApiKeyIndexView.java

```diff
@@ -77,8 +77,10 @@ private DivTag renderApiKey(
     DivTag statsDiv =
         div()
             .with(
-                p("Created " + dateConverter.formatRfc1123(apiKey.getCreateTime())),
```

Comment:
Why change the date format? it's extraneous to doing the image diffs.

Reply:
@bion Do you have any concerns changing the API UI date format?

Reply:
I changed format so that our existing logic in browser tests that hardcodes dates works well. Additionally I think the new format is more friendly:

Before: `Created Wed, 14 Dec 2022 11:11:06 -0800`
After: `Created 2022/12/14 at 11:11AM PDT`

Reply:
Friendly in terms of timezone

Reply:
Looks like an improvement to me. Thanks for surfacing @shanemc-goog !